### PR TITLE
Add Constant Moment

### PR DESCRIPTION
**Pull request by Harshp1802**

### CODE REVIEW

File: pymc/tests/test_distributions_moments.py [hunk 1]
```diff
@@ -341,3 +342,21 @@ def test_weibull_moment(alpha, beta, size, expected):
...
+        (1, 5, np.full(5, 1)),
+        (np.arange(1, 6), None, np.arange(1, 6)),
+        (
+            np.arange(1, 6),
+            (2, 5),
+            np.full((2, 5), np.arange(1,6)),
+        ),
+    ],
+)
+def test_constant_moment(c, size, expected):
```
Comment (ricardoV94) [inline pymc/tests/test_distributions_moments.py:396]: 3 of the tests should suffice for such simple distribution. Otherwise looks good :)

**Review by ricardoV94 — CHANGES_REQUESTED** [review]
See my comment above... no reason to test more than needed :)

**Comment by ricardoV94** [comment]
You will have to resolve the conflicts because of other PRs that were already merged

**Comment by Harshp1802** [comment]
> You will have to resolve the conflicts because of other PRs that were already merged

Done ✌🏻

**Comment by ricardoV94** [comment]
Seems like you have some linting issues, you may check this guide: https://github.com/pymc-devs/pymc/wiki/Python-Code-Style

**Review by ricardoV94 — APPROVED** [review]

**Comment by ricardoV94** [comment]
Thanks!

**Pull request merged**